vcbuild.bat

```diff
@@ -386,6 +393,27 @@ ssh -F %SSHCONFIG% %STAGINGSERVER% "touch nodejs/%DISTTYPEDIR%/v%FULLVERSION%/no
 if errorlevel 1 goto exit
 
 
+:build-doc
+@rem Build documentation if requested
+if not defined doc goto run
+if not exist %node_exe% (
+  echo Failed to find node.exe
+  goto run
+)
+mkdir %config%\doc
+robocopy /e doc\api %config%\doc\api
+robocopy /e doc\api_assets %config%\doc\api\assets
+
+if exist "tools\doc\node_modules\js-yaml\package.json" goto doc-skip-js-yaml
+cd tools\doc
+%npm_exe% install
+cd ..\..
```

Comment:
Not sure if I remember correctly, but for batch files, `cd` does affect the outer shell environment, right? i.e. in case of a failing `npm install` we’d be stuck here? Is that a non-issue for some reason, or am I just misremembering?